Bio/SCOP/__init__.py

```diff
@@ -594,7 +592,7 @@ def getDescendents(self, node_type):
         return nodes
 
     def getAscendent(self, node_type):
-        """ Return the ancenstor node of the given type, or None.Node type can a
+        """Return the ancenstor node of the given type, or None.Node type can a
```

Comment:
Again since your editing this docstring and have to go back and tweak this file, please split this:

```python
"""Return the ancenstor node of the given type, or None.

Node type can a two letter code or longer description. e.g. 'fa' or 'family'.
"""
```